components/script/dom/htmlmetaelement.rs

```diff
@@ -122,6 +126,22 @@ impl HTMLMetaElement {
         }
     }
 
+    /// <https://drafts.csswg.org/css-viewport/#parsing-algorithm>
+    fn apply_viewport_description(&self) {
```

Comment:
I would call this `parse_and_send_viewport_if_necessary`, because it doesn't always apply the description.